.gitignore

```diff
@@ -2,12 +2,6 @@ _build
 _opam
 .merlin
 *.install
-log-*.txt
-
 .git-ps
 
 *.cache
-
-editor/code/out
-
-result
```

Comment:
This gets produced by `nix build`.

Reply:
This belongs to the commit in #171. Only the last commit is the subject of this PR.